specification/cognitiveservices/OpenAI.Inference/models/audio.tsp

```diff
@@ -0,0 +1,206 @@
+import "@typespec/versioning";
+import "./chat.completions.tsp";
+import "./azure.cognitive.search.options.tsp";
+
+namespace Azure.OpenAI;
+
+using TypeSpec.Versioning;
+
+@doc("Defines available options for the underlying response format of output transcription information.")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+enum AudioTranscriptionFormat {
+  @doc("Use a response body that is a JSON object containing a single 'text' field for the transcription.")
+  json: "json",
+
+  @doc("""
+    Use a response body that is a JSON object containing transcription text along with timing, segments, and other
+    metadata.
+    """)
+  verbose_json: "verbose_json",
+
+  @doc("Use a response body that is plain text containing the raw, unannotated transcription.")
+  text: "text",
+
+  @doc("Use a response body that is plain text in SubRip (SRT) format that also includes timing information.")
+  srt: "srt",
+
+  @doc("""
+    Use a response body that is plain text in Web Video Text Tracks (VTT) format that also includes timing information.
+    """)
+  vtt: "vtt",
+}
+
+@doc("""
+The configuration information for an audio transcription request.
+""")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+model AudioTranscriptionOptions {
+  @doc("""
+  The audio data to transcribe. This must be the binary content of a file in one of the supported media formats:
+  flac, mp3, mp4, mpeg, mpga, m4a, ogg, wav, webm.
+  """)
+  @projectedName("csharp", "AudioData")
+  file: bytes;
+
+  @doc("""
+  The requested format of the transcription response data, which will influence the content and detail of the result.
+  """)
+  @projectedName("json", "response_format")
+  responseFormat?: AudioTranscriptionFormat;
```

Comment:
According to the swagger I believe this should default to `"json"`

`responseFormat?: AudioTranscriptionFormat = AudioTranscriptionFormat.json;`

Reply:
Thanks for raising this! Per IM conversation, it looks like we have inconsistent representation of service-side default values across the board -- a few things that are documented service-side defaults *do* encode that in TypeSpec (and the downstream generated OpenAPI v2 documents), but many more *don't*.

I filed this work item to track the holistic pass, as one or another it seems like our current treatment must be wrong:
https://github.com/Azure/azure-rest-api-specs/issues/25820

I'll leave this one as-is for now if it's not causing a problem, as we'll need to clean things up in any eventuality and "no churn" helps in the immediate term.